src/rollup.bzl

```diff
@@ -0,0 +1,23 @@
+def _rollup(ctx):
+ 
+  args = ["--config", "../../../../execroot/angular_bazel_example/rollup.config.js"]
+  args += ["--output.file", ctx.outputs.build.path]
```

Comment:
I would like to avoid path navigation like this, but I am not sure how to best navigate the folder structure or pass files along to rules.